contrib/utilities/checkdoxygen.py

```diff
@@ -49,6 +49,37 @@ def check_empty_lines_in_tables(lines):
 
     return
 
+# have more than one header with the same name in a tutorial?
+def check_multiple_defined_headers(lines):
+    headers = []
+    for l in lines:
+        # this may break if a header splits across a line
```

Comment:
Can you change this to fail with an error if the line does not contain ``<h?> ... </h?>``?

Reply:
Got it! It checks for a closing header tag on line 72 now.